bigquery/storage/managedwriter/connection.go

```diff
@@ -216,21 +285,34 @@ func (co *connection) lockingAppend(pw *pendingWrite) error {
 	var ch chan *pendingWrite
 	var err error
 
-	arc, ch, err = co.getStream(arc, false)
+	// We still need to reconnect if we need to signal a new schema for explicit streams.
+	// Rather than adding more state to the connection, we just look at the request as we
+	// do not allow multiplexing to include explicit streams.
+	forceReconnect := false
+	if !isDefaultStream(pw.writeStreamID) {
```

Comment:
seems like in the cases that `isDefaultStream` is being called, what is actually being checked is if the stream supports multiplexing. Maybe change the method name and internally for now it checks if is the default stream.

Reply:
renamed to `canMultiplex`.